wurst/_items/HealingItems.wurst

```diff
@@ -0,0 +1,47 @@
+package HealingItems
+
+import ID
+import HashMap
+import ClosureEvents
+import HealingSystem
+
+HashMap<int, real> itemSelfHealAmount = null
+HashMap<int, real> itemTargetedHealAmount = null
+
+let ABILITY_HEAL_POTION = 'A0DO'
+
+
+function initHealItemTable()
+    itemSelfHealAmount = new HashMap<int,real>()
+    itemSelfHealAmount.put(ITEM_COOKED_MEAT, 50.)
+    itemSelfHealAmount.put(ITEM_SMOKED_MEAT, 40.)
+    itemSelfHealAmount.put(ITEM_DISEASED_MEAT, 30.)
```

Comment:
Don't bother with this. It's impossible to get diseased meat without the old disease effect. Up to you if you want to handle smoked meat for now.